WtWSMS/common/culture/cultures/BP_east_germanic.txt

```diff
@@ -508,9 +508,11 @@ vidivarii = { # Migrational Era Vidivarii
 	unit_gfx = { norse_unit_gfx }
 	
 	ethnicities = {
-		33 = circumpolar_dark_hair
-		33 = circumpolar_brown_hair
-		33 = circumpolar_blonde_hair
+		#50/50 Vandal/Baltic Admixture (then added to avoid any code problems)
```

Comment:
Source for this specific origin?

Reply:
Bartosz Kontny's work on the Vidivarii. 
https://www.researchgate.net/publication/339977371_The_Making_of_the_Vidivarii_Germanic_and_Baltic_Interculturation_in_the_Late_5th_Century